packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -191,6 +191,7 @@ export class GroupedListSection extends BaseComponent<IGroupedListSectionProps,
       <div
         ref='root'
         className={ css('ms-GroupedList-group', styles.group, this._getDroppingClassName()) }
+        role='group'
```

Comment:
Rowgroup required its child element to have atleast one role="row". Internally we have designed groupedList as List with ListItem and not as row